src/advanced/merkelized-list.md

```diff
@@ -108,18 +108,31 @@ Let `T(height, index)` be a value at tree node for element `index` at height
 `height`. Elements `T(0, index)` contain serialized values of the underlying list
 according to [the Exonum binary serialization spec](../architecture/serialization.md).
 Elements `T(height, index)` for `height > 0` are hashes corresponding the following
-rules.
+rules. Each node, including root, hashes with corresponding prefix:
+
+```none
+0x0 - leaf node
```

Comment:
Nit: I find hex numbers with the odd number of digits _odd_ 🥁. IMO, `0x00` would look better.